README.md

```diff
@@ -1,31 +1,104 @@
 
-### Installing  `node`
-- Checking if node is already present in the system (should print the version number)
-    ```bash
-     node -v
-    ```
-- To install node visit [nodejs.org](https://nodejs.org/en/download/)
+Nebari is an opinionated open source JupyterHub distribution with a variety of common integrations to help you and your team get set up with all your favorite tools ...  and a few you didn't know you needed! 
 
-### Building the site
+Check out our full [documentation](https://nebari-docs.netlify.app/)!
 
-Install the neccesary packages
-```bash
-cd docs
-npm install
-```
+---
+
+## Table of contents
+
+- [Nebari 101](#nebari-101)
+- [Nebari on-prem](#nebari-on-prem)
+  <!-- - [:cloud: Cloud Providers](#cloud-cloud-providers) -->
+- [:computer: Installation](#computer-installation)
+- [:question: Questions?](#question-questions)
+- [:book: Code of Conduct](#book-code-of-conduct)
+- [:gear: Installing the Development version of Nebari](#gear-installing-the-development-version-of-nebari)
+- [:raised_hands: Contributions](#raised_hands-contributions)
+- [License](#license)
+
+## Nebari 101
+
+Nebari Cloud deployments use Kubernetes and are is built using [Terraform](https://www.terraform.io/), [Helm](https://helm.sh/), and
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions). Terraform handles the build, change, and versioning of the infrastructure. Helm helps to define, install,
+and manage [Kubernetes](https://kubernetes.io/ "Automated container deployment, scaling, and management"). GitHub Actions is used to automatically create commits when the
+configuration file (`nebari-config.yaml`) is rendered, as well as to kick off the deployment action.
+
+Nebari aims to abstract all these complexities for its users. Hence, it is not necessary to know any of the above mentioned technologies to have your project successfully deployed.
+
+> TLDR: If you know GitHub and feel comfortable generating and using API keys, you should have all it takes to deploy and maintain your system without the need for a dedicated
+> DevOps team. No need to learn Kubernetes, Terraform, or Helm.
+
+## Nebari on-prem
+
+We also have a version of Nebari based on OpenHPC. It can be used to deploy onto HPC systems or on bare-metal. 
+
+> NOTE: This tool is currently under development. Curious? Check out the [Nebari HPC](https://github.com/Quansight/qhub-hpc) repository.
+
+
+<!-- ### :cloud: Cloud Providers
+
+Nebari offers out-of-the-box support for [Digital Ocean](https://www.digitalocean.com/), Amazon [AWS](https://aws.amazon.com/),
+[GCP](https://cloud.google.com/ "Google Cloud Provider"), and Microsoft [Azure](https://azure.microsoft.com/en-us/).
+
+![High-level illustration of Nebari architecture](docs/source/images/qhub-cloud_architecture.png)
+
+For more details, check out the release [blog post](https://www.quansight.com/post/announcing-qhub). --> 
```

Comment:
I didn't think the cloud provider details needed to be included in this high level readme. If you don't agree, I can uncomment.

Reply:
Agree - let's delete